robot_name: newton

motors_params:
  # HAA : Hip Abduction/Adduction
  FL_HAA:
    enabled: False ## enabled on the can bus
    name: FL_HAA
    node_id: 0
    position_limit: 0.5
    direction: -1
    gear_ratio: 9.0
    start_pos: 3.14
  
  
  FL_HFE:
    enabled: True
    name: FL_HFE
    node_id: 0
    position_limit: 1.57
    direction: -1
    gear_ratio: 9.0
    start_pos: 1.57

  FL_KFE:
    enabled: True
    name: FL_KFE
    node_id: 1
    position_limit: 3
    direction: -1
    gear_ratio: 9.0
    start_pos: 3.14

  FR_HAA:
    enabled: False
    name: FR_HAA
    node_id: 3
    position_limit: 0.5
    direction: -1
    gear_ratio: 9.0
    start_pos: 0.0

  FR_HFE:
    enabled: True
    name: FR_HFE
    node_id: 2
    position_limit: 1.57
    direction: 1
    gear_ratio: 9.0
    start_pos: 1.57
  
  FR_KFE:
    enabled: True
    name: FR_KFE
    node_id: 3
    position_limit: 3
    direction: 1
    gear_ratio: 9.0
    start_pos: 3.14

  HL_HAA:
    enabled: False
    name: HL_HAA
    node_id: 6
    position_limit: 0.5
    direction: -1
    gear_ratio: 9.0
    start_pos: 0.0

  HL_HFE:
    enabled: True
    name: HL_HFE
    node_id: 4
    position_limit: 1.57
    direction: -1
    gear_ratio: 9.0
    start_pos: -1.57

  HL_KFE:
    enabled: True
    name: HL_KFE
    node_id: 5
    position_limit: 3
    direction: -1
    gear_ratio: 9.0
    start_pos: 3.14


  HR_HAA:
    enabled: False
    name: HR_HAA
    node_id: 9
    position_limit: 0.5
    direction: -1
    gear_ratio: 9.0
    start_pos: 0.0

  HR_HFE:
    enabled: True
    name: HR_HFE
    node_id: 6
    position_limit: 1.57
    direction: 1
    gear_ratio: 9.0
    start_pos: -1.57

  HR_KFE:
    enabled: True
    name: HR_KFE
    node_id: 7
    position_limit: 3
    direction: -1
    gear_ratio: 9.0
    start_pos: 3.14
